### ISSUE TITLE
Alphabetically sort language list

### ISSUE THREAD

**Issue by Lod3** [kaixxx/noScribe#94]
The ability to go through the very extensive list of supported languages (in alphabetical order) faster would save time and imporve useability.
Currently the list is not sorted, and this makes finding a specific language quite cumbersome. 
If I can suggest sorting the list alphabetically based on the full written english name rather than sorting it on the abbreviation which requires knowledge on how the language name is abbreviated. 

Thank you again, for this tool.

**Comment by kaixxx** [comment]
Yes, that's a good suggestion. I'll put it on my list for the next version. Thanks!
